test/test-signatures.spec.ts

```diff
@@ -30,11 +33,17 @@ describe("Signatures", function () {
     testOneResponse = (
       pathToXml: string,
       shouldErrorWith: string | false,
-      amountOfSignaturesChecks: number | undefined
+      amountOfSignaturesChecks: number | undefined,
+      options?: Partial<SamlOptions>
```

Comment:
Why not use the type SamlConfig instead as that better represents what a consumer should be using.

Reply:
Good suggestion. I've updated the code.